Tests/travis/README_tabs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+functions="$(dirname $0)/functions.sh"
+if [ -f "$functions" ]; then
+  . "$functions"
+else
+  echo "functions file '$functions' not found!"
+  exit 1
+fi
+
+e=0
+while read f; do
+  [ -z "$f" ] && continue
+  r=`getAddedLines "$f" | grep -P '^.*?\t+.*'`;
```

Comment:
Won\`t `grep -P '\t'` work just as well?

Reply:
Thank you, regexp was simplified.